sonar-java-plugin/src/main/resources/org/sonar/l10n/java/rules/java/Sonar_way_profile.json

```diff
@@ -498,6 +498,7 @@
     "S6856",
     "S6857",
     "S6862",
-    "S6863"
+    "S6863",
+    "S6891"
```

Comment:
Do we want this in Sonar way after all?

Reply:
Yes, you're right, we don't want it. Fixed it in this file and also in the RSPEC.